### ISSUE TITLE
Errors occurred using docker on MacOX Catalina 10.15.5 

### ISSUE THREAD

**Issue by chihwan** [ncbi/pgap#99]
Hello guys. 
I have tried to use the latest PGAP docker image ncbi/pgap:2020-07-09.build4716 on MACOX. 
The docker desktop version is 2.3.03(45519) and docker environments were allocated with 1 CPU and 2GB memory. Actually I tested several times in different environments such as 10 CPUs with 2GB memory(it's the default), 4 CPUs with 8GB memory, 2CPUs with 4GB memory. But all the tests were failed in the same step "[job cluster_blastp_wnode] completed permanentFail"(I guessed). I'm not sure which step is the primary error because so many errors occurred during analysis.  So, I attached the log files 1 CPU with 2GB RAM [cwltool.log](https://github.com/ncbi/pgap/files/5166204/cwltool.log), 2 CPUs with 4GB RAM
[cwltool.log](https://github.com/ncbi/pgap/files/5166317/cwltool.log). Please see the log file and let me know what is the problems.  

The snippet of error log is like below.

```
[2020-09-03 04:10:26] INFO [job cluster_blastp_wnode] Max memory used: 52MiB
[2020-09-03 04:10:27] WARNING [job cluster_blastp_wnode] completed permanentFail
[2020-09-03 04:10:27] WARNING [step cluster_blastp_wnode] completed permanentFail
[2020-09-03 04:10:27] INFO [workflow cluster_and_qdump] completed permanentFail
[2020-09-03 04:10:27] WARNING [step cluster_and_qdump] completed permanentFail
[2020-09-03 04:10:27] INFO [workflow Find_Naming_Protein_Hits_I] completed permanentFail
[2020-09-03 04:10:27] WARNING [step Find_Naming_Protein_Hits_I] completed permanentFail
[2020-09-03 04:10:27] INFO [workflow bacterial_annot_2] completed permanentFail
[2020-09-03 04:10:27] WARNING [step bacterial_annot_2] completed permanentFail
[2020-09-03 04:10:27] INFO [workflow standard_pgap] completed permanentFail
[2020-09-03 04:10:27] WARNING [step standard_pgap] completed permanentFail
[2020-09-03 04:10:27] INFO [workflow ] completed permanentFail
{
    "calls": {
        "location": "file:///pgap/output/calls.tab",
        "basename": "calls.tab",
        "class": "File",
        "checksum": "sha1$da39a3ee5e6b4b0d3255bfef95601890afd80709",
        "size": 0,
        "path": "/pgap/output/calls.tab"
    },
    "final_asndisc_error_diag": null,
    "final_asnval_error_diag": null,
    "gbk": null,
    "gff": null,
    "initial_asndisc_error_diag": {
        "location": "file:///pgap/output/initial_asndisc_diag.xml",
        "basename": "initial_asndisc_diag.xml",
        "class": "File",
        "checksum": "sha1$da39a3ee5e6b4b0d3255bfef95601890afd80709",
        "size": 0,
        "path": "/pgap/output/initial_asndisc_diag.xml"
    },
    "initial_asnval_error_diag": {
        "location": "file:///pgap/output/initial_asnval_diag.xml",
        "basename": "initial_asnval_diag.xml",
        "class": "File",
        "checksum": "sha1$da39a3ee5e6b4b0d3255bfef95601890afd80709",
        "size": 0,
        "path": "/pgap/output/initial_asnval_diag.xml"
    },
    "input_fasta": {
        "class": "File",
        "location": "file:///pgap/output/ASM2732v1.annotation.nucleotide.1.fasta",
        "size": 588482,
        "basename": "ASM2732v1.annotation.nucleotide.1.fasta",
        "checksum": "sha1$f6129783cc8562db7bca3c87310d57d8dd07ce2c",
        "path": "/pgap/output/ASM2732v1.annotation.nucleotide.1.fasta"
    },
    "input_submol": {
        "class": "File",
        "location": "file:///pgap/output/pgap_submol_l69wv6tk.yaml",
        "size": 1667,
        "basename": "pgap_submol_l69wv6tk.yaml",
        "checksum": "sha1$c76170f856551e6f31a13cd29e4ab394c153385f",
        "path": "/pgap/output/pgap_submol_l69wv6tk.yaml"
    },
    "nucleotide_fasta": null,
    "protein_fasta": null,
    "sqn": null
}
[2020-09-03 04:10:27] WARNING Final process status is permanentFail
```

**Comment by chihwan** [comment]
After reading the closed issues #26, #27 (it is the same I think), I changed the docker desktop environment to 8 CPUs with 16.75GB RAM.  Then PGAP is successfully executed, and I got the final results. Logfile [cwltool.log](https://github.com/ncbi/pgap/files/5172665/cwltool.log) is here. So, it does not matter for now. Should I delete this issue?

F.Y.I. I tested with ubuntu 18.04 and 16.04 on docker. There are permission errors of GENEMARK included in the supplemental reference data "input-2020-07-09.build4716". So I change the permission like below.
`cd input-2020-07-09.build4716/GeneMark/API_2.2;
chmod a+rwx compp gmhmmp gmsn.pl biogen gmhmmp2 gms2.pl;
`

Thank you.
